src/components/application_manager/include/application_manager/commands/mobile/alert_maneuver_request.h

```diff
@@ -83,6 +83,21 @@ class AlertManeuverRequest : public CommandRequestImpl {
    */
   bool PrepareResponseParameters(mobile_apis::Result::eType& result_code,
                                  std::string& return_info);
+
+  /**
+   * @brief Checks result code from HMI for splitted RPC
+   * and returns parameter for sending to mobile app.
+   * @param navigation_alert_info contains result_code from HMI response and
+   * interface that returns response
+   * @param tts_alert_info contains result_code from HMI response and
+   * interface that returns response
+   * @return true if result code complies successful result code
+   * otherwise returns false
+   */
+  bool PrepareResultForMobileResponse(
+      ResponseInfo& navigation_alert_info,
+      ResponseInfo& tts_alert_info) const OVERRIDE;
```

Comment:
@VProdanov fixed in https://github.com/smartdevicelink/sdl_core/pull/1567/commits/20a8cd4c0ffd1f7243cf249bf341f18b60787b24